Makefile

```
@@ -0,0 +1,19 @@
+build: migrate.up
+	go build main.go
+
+migrate.up:
+	cp .env.example .env
+	cd db
+        include .env
+	DB_MIGRATIONS_DIR = $(PWD)/db/migrations
+	EXPORT SQLITE_DSN
+	
+	goose -dir  $(DB_MIGRATIONS_DIR) sqlite3 $(SQLITE_DSN) up
+	cd ..
+
+migrate.down:
+	goose down
```

this would probably be affected as well
```suggestion
	goose -dir $(DB_MIGRATIONS_DIR) sqlite3 $(SQLITE_DSN) down
```

---

Thanks.